src/components/button/button.svelte

```diff
@@ -294,15 +294,25 @@
     &:disabled:not(.isLoading) {
       --color: var(--leo-color-text-primary);
     }
+
+    &.fab {
+      --padding-y: 0;
+      --padding-x: 0;
+    }
   }
   .leoButton.isPlainFaint {
     --radius: 8px;
     --padding-x: 2px;
-    --color: var(--leo-color-text-secondary);
+    --color: var(--leo-button-faint-color, var(--leo-color-text-secondary));
     --color-hover: var(--leo-color-gray-70);
     --box-shadow-hover: none;
-    --icon-color: var(--leo-color-icon-default);
+    --icon-color: var(--leo-button-faint-color, var(--leo-color-icon-default));
     --icon-hover-color: var(--leo-color-gray-70);
+
+    &.fab {
+      --padding-y: 0;
+      --padding-x: 0;
+    }
```

Comment:
The padding for icon only, plain faint buttons is different than the others, according to Figma.